### ISSUE TITLE
Add option to hide backdrop

### ISSUE THREAD

**Issue by raptormama** [maxheyn/TorghastTeams#6]
Fantastic addon, especially since the default quest UI doesn't show powers for me - I'm left-eye dominant, so I've moved the quest list to the left side of the screen and the powers window opens off screen as a result. So TorghastTeams allows me to display my powers in a separate place that does work for me.

But I'd love to be able to hide the backdrop so the display doesn't take up so much screen space. It would also be beneficial to be able to move the display for each player separately, though I know that might be harder to implement.

(For what it's worth, I use ElvUI. TGT nicely picks up on the ElvUI interface font and works fine with it.)

**Comment by maxheyn** [comment]
First of all, thank you so much for bringing up an issue here on GitHub! It means a lot to me!

> TorghastTeams allows me to display my powers in a separate place that does work for me.

Thank you! This makes me happy to see! 😊

> I'd love to be able to hide the backdrop so the display doesn't take up so much screen space. It would also be beneficial to be able to move the display for each player separately...

Hear you loud and clear, we're going to be looking at that soon. The hiding/opacity part might be a bit tricky but I'm sure we'll be able to figure something out to make that work. 😀

Thanks again for the feedback!

**Comment by maxheyn** [comment]
Thank you for your suggestion! We have come up with a solution here: a5c734976d261d36d305fd3053bcd6d18f96eb92

When you are in Torghast, Ctrl+Left Click the minimap button and you will be able to switch to a less obtrusive UI where you can move the Anima Power Displays around as you please.

Please let us know what you think! Thanks! 😊

**Comment by raptormama** [comment]
I like it! It's just what I'm hoping for, and I'm including a pair of screenshots. (They also happen to show why the default power UI opens off-screen in my setup).

At first, I got both the old and the new TGT UI -- I'd expected that the UI would drop the background and be in the same place (I manually had dragged the window to the upper right of the screen). I managed to get the "old" window to go away after a few more clickings of the minimap icon. I'm not sure if I was in combat at the time I clicked or not (I wasn't in either of these screenshots).

![Image 1](https://user-images.githubusercontent.com/5473082/103068586-0f05c200-4583-11eb-8fe9-380e68fecbe4.png)

After a few false starts trying to get the windows to move (it seems that a player has to be in the instance and the name 'header' has to not be gray), I got them to go to where I want to keep them.

![image 2](https://user-images.githubusercontent.com/5473082/103068671-3d839d00-4583-11eb-9a4f-b063a6522719.png)

Alas, currently, it doesn't look like the addon remembers where I put them. We ran two wings today, and when we entered the second one, the windows were in the default center location.

In my case, I like to put my own next to the right-side chat window, since there'll always be a box for me. And I put my teammates below the minimap, so in this case I'd want the 'first' player to be the top right one of those four, followed by the bottom right, then the top left, then the bottom left.

Since they defaulted to the middle of the screen, I had to move them out of the way before starting the run.

The appearance, though, is exactly what I wanted. And I'm surprised it got added so fast! I can tell this is a passion project for you. Thank you so much for working on it during the holiday week! I'd never have asked that of anyone.

**Comment by maxheyn** [comment]
I'm glad you like it! Thanks a ton for the screenshots by the way. I'm not sure why in the first one you were able to open up both the new "simple mode" UI and both the old one, that didn't happen in our testing. Did you open the default UI with the background with the minimap button or with the chat command `/tgt show`? That might have been it but I'm not able to test it at this time, I'll look into it next time I can.

We've updated it a tiny bit since then so now you should be able to move the frames even if you and your teammates haven't picked up anima powers yet, but haven't gotten around to saving the position yet, so at the moment you'll still have to move things around every time you start a new run, sorry!

> The appearance, though, is exactly what I wanted. And I'm surprised it got added so fast! I can tell this is a passion project for you. Thank you so much for working on it during the holiday week! I'd never have asked that of anyone.

It really is a passion project, I'm so glad you recognize that! 😊 It feels good to be recognized for hard work, and I really am glad that you like it so far! I'm working almost every day to fix things and improve it. If you have any more suggestions, feel free to open more issues. I'd love to hear anything!

**Comment by raptormama** [comment]
So! An update. It looks like the button now seems to correctly toggle between the different UI modes -- I couldn't get it to double up again. (When I loaded in, I tried starting a video capture to catch the weird behavior for sharing it here).

Dragging the window to where I want it does seem to be more reliable this time around, and I noticed that the power circles now automatically scale down as more get added, which I like.

~~Along with remembering window placement, I'd love to see the addon remember whether it was turned 'on' or not (visible) at the end of the last run, and which mode it was in. Bonus cherry-on-top: ElvUI skinning of the low-profile mode~~. (Moved these to new Suggestions).

It's looking really good now!

Have a great New Year. :)
